grammar/terraform_plan.treetop

```diff
@@ -77,7 +77,20 @@ grammar TerraformPlan
   rule attribute
     attribute_name:[^:]* ':' _? attribute_value:[^\n]+ {
       def to_ast
-        { attribute_name.text_value => attribute_value.text_value }
+        { attribute_name.text_value => sanitize_value }
+      end
+
+      def sanitize_value
+        val = attribute_value.text_value
+
+        # We'll perform some sanitization of the values within the parser.
+
+        # With Terraform >= 0.10.4, the <computed> field is now without quotes,
+        # which will cause problem with how downstream we process the output.
+        # Convert it to have quotes and handle < 0.10.4 as well.
+        val = val.gsub(%r{<computed>}, '"<computed>"').gsub(%r{""<computed>""}, '"<computed>"')
```

Comment:
There's a part of me that wonders what happens when a user literally uses the string `<computed>` as a value for the attribute (which is the scenario that likely prompted them to change the output format in the first place).

Alternatively, if the user as an attribute like `"Hello my <computed> value"`, the output will be "Hello my \"<computed>\" value"`, which is probably not what we want.

I believe I've taken care of this in c19e965ee07a1fa1ec268bfde4c9890b86cb5764.